client/src/actions/index.js

```diff
@@ -161,7 +161,7 @@ const dispatchDiffExpErrors = (dispatch, response) => {
   }
 };
 
-const requestDifferentialExpression = (set1, set2, num_genes = 10) => async (
+const requestDifferentialExpression = (set1, set2, num_genes = 50) => async (
```

Comment:
This is a big UI change (I think -- assuming it now displays top 50 genes, rather than top 10).  

@signechambers1 - you good with this?

@colinmegill - in the future, could we break unrelated stuff like this into its own PR?